v2/utils.go

```diff
@@ -167,7 +148,7 @@ func parseCgroupFromReader(r io.Reader) (GroupPath, error) {
 		}
 		// text is like "0::/user.slice/user-1001.slice/session-1.scope"
 		if parts[0] == "0" && parts[1] == "" {
-			return GroupPath(parts[2]), nil
```

Comment:
still feel that it is not more explicit. the pattern is `0::PATH` and not need to split it into 3 parts😂